static/bidder-info/bliink.yaml

```diff
@@ -15,6 +15,9 @@ capabilities:
       - video
       - native
 userSync:
+  iframe:
+    url: "https://tag.bliink.io/usersync.html?gdpr={{.GDPR}}&gdprConsent={{.GDPRConsent}}&uspConsent={{.USPrivacy}}&redirect={{.RedirectURL}}"
+    userMacro: "$UID"
```

Comment:
@LouisBillaut 
iframe url doesn't  redirect back 

https://tag.bliink.io/usersync.html?gdpr=&gdprConsent=&uspConsent=&redirect=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dbliink%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID

Reply:
> @LouisBillaut iframe url doesn't redirect back with UID
> 
> https://tag.bliink.io/usersync.html?gdpr=&gdprConsent=&uspConsent=&redirect=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dbliink%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID

@LouisBillaut  requesting to address comment

Reply:
@onkarvhanumante Hi, the redirect is loaded directly into the iframe

Reply:
> @onkarvhanumante Hi, the redirect is loaded directly into the iframe is it ok ?

UUID was not populated in rediect url inside iframe html tag.

Reply:
Hi  @onkarvhanumante, we have solved the issue. Is it ok now ?

Reply:
> https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dbliink%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID

@LouisBillaut  tested out above url. UID is not replaced in redirect url. Could you check above url

Reply:
@LouisBillaut  requesting to address above comment

Reply:
> @LouisBillaut requesting to address above comment

Hi @onkarvhanumante,
we've done a fix, now it should work.

Reply:
> > @LouisBillaut requesting to address above comment
> 
> Hi @onkarvhanumante, we've done a fix, now it should work.

fix seems to be working